packages/notifi-react-card/lib/assets/CheckIcon.tsx

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 
-export type Props = React.SVGProps<SVGSVGElement>;
+export type Props = Readonly<{
+  className?: string;
+}>;
```

Comment:
hmmm i think in general it's nice to be able to override other props, like width and height

Reply:
Modified: Allow to pass in svg props into CheckIcon
```
export type Props = Readonly<
  {
    className?: string;
  } & React.SVGProps<SVGSVGElement>
>;
```